docs/element plus.md

```
@@ -0,0 +1,46 @@
+# Element Plus组件物料文档
+
+## 使用手册
+
+### 自动化脚本使用
+
+在项目根目录打开shell，执行下面命令生成物料文件：
+```shell
+node ./scripts/genMaterial.js
+```
+![](./images/01.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)